utils/vaults.json

```diff
@@ -1152,4 +1152,16 @@
     "VAULT_STATUS": "new",
     "CHAIN_ID": 1
   }
```

Comment:
a comma is missing here

Reply:
you can use the script here: https://github.com/saltyfacu/ape-tax#add-an-experimental-experiment